python/tvm/autotvm/task/topi_integration.py

```diff
@@ -231,14 +239,27 @@ def _decorate(topi_schedule):
         @_register_task_schedule(task_name)
         def wrapper(outs, *args, **kwargs):
             """wrapper function for topi schedule"""
-            workload = get_workload(outs, task_name)
+            task_env = TaskExtractEnv.current
+            if GLOBAL_SCOPE.tune_subgraph and task_env is not None and not task_env.tracing:
+                # get workload of subgraph
```

Comment:
May add an explanation for this `get_workload_post_order(...)`, why necessary?

Reply:
have added an annotation in code -> get subgraph's workload which is attached to the last op, because the anchor op has attached workload too, so we should get by post order